addons/account/wizard/account_resequence.py

```diff
@@ -52,7 +52,7 @@ def _compute_first_name(self):
         self.first_name = ""
         for record in self:
             if record.move_ids:
-                record.first_name = min(record.move_ids._origin.mapped('name'))
+                record.first_name = min(name if name else "" for name in record.move_ids._origin.mapped('name'))
```

Comment:
you can push everything in the `mapped` i.e.
```python
min(record.move_ids._origin.mapped(lambda move: move.name or ""))
```

Reply:
thanks for the tip